core/domain/html_validation_service.py

```diff
@@ -831,3 +831,69 @@ def get_filename_with_dimensions(old_filename, exp_id):
     new_filename = regenerate_image_filename_using_dimensions(
         old_filename, height, width)
     return new_filename
+
+
+def add_image_id_and_remove_filepath_from_image_tag(
+        html, image_info_dict_1, image_info_dict_2):
+    """Adds image_id-with-value in image tag.
+
+    Args:
+        html: str. HTML of content in which image tag present.
+        image_info_dict_1: dict. Dict contaning image id with source, image ids
+            that needs to be added in image tag.
+        image_info_dict_2: dict. Dict contaning image ids with source, image ids
+            that has already been added in image tags.
+
+    Returns:
+        html: str. HTML contains image tag with image id and without filepath.
+    """
+    soup = bs4.BeautifulSoup(html.encode('utf-8'), 'html.parser')
```

Comment:
@nithusha21, can you please review this method? (is it implemented in the way you and Keshav have discussed?)

Reply:
ping @nithusha21